content/doc/book/managing/system-properties.adoc

```diff
@@ -489,6 +770,23 @@ dd {
     **Since:** 2.93 +
     **Description:** True to use native (JNA/JNR) implementation to set file permissions instead of NIO
 
+`hudson.WebAppMain.forceSessionTrackingByCookie`::
+    **Default:** `true` +
+    **Since:** 2.234 +
+    **Description:** Set to `false` to not force session tracking to be done via cookie.
+    Escape hatch for https://issues.jenkins-ci.org/browse/JENKINS-61738[JENKINS-61738].
+
+`hudson.widgets.HistoryWidget.threshold`::
+    **Default:** 30 +
+    **Since:** 1.433 +
+    **Description:** How many builds to show in the build history side panel widget.
+
+`HUDSON_HOME`::
```

Comment:
Time to introduce a section for the deprecated properties. Not needed for this pull request

Reply:
On my todo list.